src/entity/Game.ts

```diff
@@ -1,45 +1,43 @@
-import { BigIntType, Collection, Entity, Enum, ManyToOne, OneToMany, PrimaryKey, Property, QueryOrder, Ref, Reference } from '@mikro-orm/core';
-import { Card, Player, Tournament } from './index.js';
-import { GameRepository } from '../repository/index.js';
-
-// @Entity({ customRepository: () => GameRepository })
-@Entity()
-export class Game {
-
-    @PrimaryKey({ type: BigIntType })
-        id!: number;
-
-    @ManyToOne(() => Card, { ref: true, name: 'startcard'})
-        startcard: Ref<Card>;
-
-    @Property({ onCreate: () => new Date() })
-        gameStartedAt!: Date;
-
-    @Property({ nullable: true })
-        gameEndedAt?: Date;
-
-    @OneToMany(() => Player, player => player.game, { orderBy: { playerIndex: QueryOrder.ASC } })
-        players = new Collection<Player>(this);
-
-    @Property({ nullable: true })
-        winner?: Ref<Player>;
-
-@ManyToOne(() => Tournament, { ref: true, nullable: true })
-    tournament?: Ref<Tournament>;
-
-    @Property({ nullable: true })
-@Enum(() => GameRole)
-        role?: GameRole;
-
-    @Property({ nullable: true })
-        encounterRound?: number;
-
-    constructor(startcardID: number) {
-        this.startcard = Reference.createFromPK(Card, startcardID);
-    }
-}
-
-export enum GameRole {
-    ROUND_ROBIN,
-    SUDDEN_DEATH
-}
+import { BigIntType, Collection, Entity, Enum, ManyToOne, OneToMany, PrimaryKey, Property, QueryOrder, Ref, Reference } from '@mikro-orm/core';
+import { Card, Player, Tournament } from './index.js';
+
+@Entity()
+export class Game {
+
+    @PrimaryKey({ type: BigIntType })
+        id!: number;
+
+    @ManyToOne(() => Card, { ref: true, name: 'startcard'})
+        startcard: Ref<Card>;
+
+    @Property({ onCreate: () => new Date() })
+        gameStartedAt!: Date;
+
+    @Property({ nullable: true })
+        gameEndedAt?: Date;
+
+    @OneToMany(() => Player, player => player.game, { orderBy: { playerIndex: QueryOrder.ASC } })
+        players = new Collection<Player>(this);
+
+    @Property({ nullable: true })
+        winner?: Ref<Player>;
+
+@ManyToOne(() => Tournament, { ref: true, nullable: true })
+    tournament?: Ref<Tournament>;
+
+    @Property({ nullable: true })
+@Enum(() => GameRole)
+        role?: GameRole;

```

Comment:
Einrückung etwas komisch...
```suggestion
    @ManyToOne(() => Tournament, { ref: true, nullable: true })
        tournament?: Ref<Tournament>;

    @Property({ nullable: true })
    @Enum(() => GameRole)
        role?: GameRole;

```